src/bin/cp.ts

```diff
@@ -139,7 +139,10 @@ function main (): void {
 	let dest: string = args[--outstanding];
 
 	fs.stat(dest, function (oerr: any, stats: fs.Stats): void {
-		if (oerr) {
+		if (dest[dest.length - 1] === '/' && (oerr || (stats && !stats.isDirectory()))) {
```

Comment:
This is almost perfect -- I think the `dest[dest.length - 1] === '/'`, as `dest` may be a directory even if the user didn't specify a trailing slash